cli/src/main/java/org/infinispan/cli/commands/rest/Migrate.java

```diff
@@ -111,6 +111,27 @@ public boolean isHelp() {
       }
    }
 
+
+   @CommandDefinition(name = "source-connection", description = "Obtains the remote store configuration if a cache was previously connected to another cluster")
```

Comment:
Is this description correct? Isn't the behaviour that the remote store configuration is returned if the cache is currently connected to another cluster?

Reply:
Yes, that's what I meant, I will remove the "was previously" part.